CMakeLists.txt

```diff
@@ -46,6 +47,23 @@ add_subdirectory(demos)
 add_subdirectory(benchs)
 add_subdirectory(tutorial/cpp)
 
+if (FAISS_ENABLE_JEMALLOC)
+  # Use jemalloc for memory allocation if FAISS_ENABLE_JEMALLOC is set.
+  # In case faiss is a sub-project of another project,
+  # the values to following cmake variables must be set by the parent project:
+  # JEMALLOC_INCLUDE_DIR -> where to find jemalloc/jemalloc.h
+  # JEMALLOC_LIBRARIES -> where to find jemalloc library
+  # Check if the parent project has set the required variables and if not, find jemalloc, using find_package.
+message(STATUS "FAISS JEMALLOC library: ${JEMALLOC_LIBRARIES}")
```

Comment:
Seems weird to print these out twice. Also, indenting is wrong here.

Reply:
sorry there is a build failure in linux where the if condition is failing,
FAISS JEMALLOC library: optimized;/opt/couchbase/lib/libjemalloc.so;debug;/opt/couchbase/lib/libjemallocd.so

i think the EXISTS clause is failing for this value of the variable